README.md

```diff
@@ -102,13 +107,30 @@ use the const `COMMON_CONTEXT` to make some variables available for all template
     send_notification("foo@bar.com", "event_created", context, [attachment])
     
     ```
-   
+
 8. By default, notifications will be sent immediately, if you only want to add notification to the message queue
  and send it later, configure `ILMOITIN_QUEUE_NOTIFICATIONS`:
     ```python
     ILMOITIN_QUEUE_NOTIFICATIONS = True
     ```
 
+## Using the GrapQL API
+1. Add the `Query` to the entrypoint where you build your schema:
```

Comment:
should we enumerate the steps if there is only 1? 🙂

Reply:
I think the first step is installing the extras, and then this is the second step. maybe put them together in README 🙂